examples/test_method/Screen_set_colors.rb

```diff
@@ -1,34 +1,32 @@
 #!/usr/bin/env ruby
 
 require 'rubygems'
-require "newt"
+require 'newt'
```

Comment:
I am really a hater of this cop, IMHO it is really not worth any effort and it only causes pain when merging/rebasing. But since you already did a huge amount of work, I am okay with this.